test/celeritas/GlobalTestBase.hh

```diff
@@ -139,7 +138,32 @@ class GlobalTestBase : public celeritas_test::Test
     SPConstRng         rng_;
     SPConstCore        core_;
     SPOutputManager    output_;
+};
 
+//---------------------------------------------------------------------------//
+/*!
+ * Reuse geometry across individual tests.
+ *
+ * This is helpful for slow geometry construction or if the geometry has
+ * trouble building/destroying multiple times per execution due to global
+ * variable usage (VecGeom, Geant4).
+ *
+ * The "geometry basename" should be the filename without extension of a
+ * geometry file inside `test/celeritas/data`.
+ */
+class GlobalGeoTestBase : virtual public GlobalTestBase
```

Comment:
Why not creating a separate file for `GlobalGeoTestBase`?

Reply:
You got me, I was being lazy 😆